src/main/java/com/minecolonies/core/entity/pathfinding/pathjobs/AbstractPathJob.java

```diff
@@ -551,7 +551,7 @@ protected double computeCost(
             }
             else if (pathingOptions.dropCost != 1)
             {
-                cost *= pathingOptions.dropCost * Math.abs(dPos.getY());
+                cost *= pathingOptions.dropCost * Math.abs(dPos.getY() * dPos.getY());
```

Comment:
could we except stair blocks from this?

Reply:
stair blocks are always max 1 diff, this only affects jumping multiple blocks